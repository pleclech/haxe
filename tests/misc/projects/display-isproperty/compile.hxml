-main Main
--display Main.hx@130