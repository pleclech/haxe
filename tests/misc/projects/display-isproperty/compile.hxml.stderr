<list>
<i n="bar" ip="1"><t>Int</t><d></d></i>
<i n="foo" ip="0"><t>Bool</t><d></d></i>
<i n="get_bar" ip="0"><t>Void -&gt; Unknown&lt;0&gt;</t><d></d></i>
</list>

